youtube_dl/extractor/peertube.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from ..compat import compat_urlparse
+from .common import InfoExtractor
+
+
+class PeertubeIE(InfoExtractor):
+    IE_DESC = 'Peertube Videos'
+    IE_NAME = 'Peertube'
+    _VALID_URL = r'https?:\/\/peertube\.touhoppai\.moe\/videos\/watch\/(?P<id>[0-9|\-|a-z]+)'
```

Comment:
Nothing changed.

Reply:
What is exactly wrong with the capture group as it seems correct to me and runs without any errors?